packages/rs-dpp/src/data_contract/data_contract.rs

```diff
@@ -388,22 +401,51 @@ impl DataContract {
             .map(|(doc_type, schema)| (String::from(doc_type), get_binary_properties(schema)))
             .collect();
     }
+
+    pub fn get_identifiers_and_binary_paths(&self, document_type: &str) -> (Vec<&str>, Vec<&str>) {
+        let maybe_binary_properties = self.get_binary_properties(document_type);
+        let mut binary_paths: Vec<&str> = vec![];
+        let mut identifiers_paths: Vec<&str> = vec![];
+
+        if let Ok(binary_properties) = maybe_binary_properties {
```

Comment:
Won't it result in suppressing the error, or we don't care for this particular case?

Reply:
In this case we shouldn't care IMO. I've made an additional comment, to mark that this is intentinal.